lib/jekyll-feed/feed.json

```diff
@@ -0,0 +1,72 @@
+{
+  "version": "https://jsonfeed.org/version/1",
+  {% assign site_title = site.title | site.name %}
+  {% if site_title %}
+    "title": {{ site_title | smartify | json }},
```

Comment:
i can't find `json` filter there https://jekyllrb.com/docs/templates/ may be `jsonify`